RedHat-UX/red-hat-design-system
# docs: fix file watch for element docs

Fixes #907

## What I did

1. manually calculate element docs dirs to add to 11ty watch


## Testing Instructions

1. `npm ci && npm run build && npx eleventy --watch --serve`
2. edit something in `elements/*/docs/*` and see that 11ty rebuilds
